fullstack/pom.xml

```diff
@@ -24,6 +24,7 @@
         <awaitility.version>1.7.0</awaitility.version>
         <commons-io.version>2.5</commons-io.version>
         <commons-lang.version>3.4</commons-lang.version>
+        <ehcache3.version>3.2.0</ehcache3.version>
```

Comment:
is it overriding a spring boot property ?
why isn't spring boot using this version in the first place ?